### DISCUSSION TITLE
New style for MessageButton

### DISCUSSION THREAD

**Discussion by Aymeric-Tremoulet** [discordjs/discord.js#7453] [Ideas]
Hello,

Since we've got "DANGER" and "SUCCESS", I really wish we had a "WARNING" style for MessageButton.

I guess it would be yellow/orange.

Thanks for your time

**Comment by KhafraDev** [comment]
Discord only has a few button types/"colors" that you can choose. You can view them [here](https://discord.com/developers/docs/interactions/message-components#button-object-button-styles).

**Comment by vladfrangu** [comment]
We're not Discord, and we cannot control what button styles there are! You should suggest it to Discord over at their [Feedback portal](https://support.discord.com/hc/en-us/community/topics)!
